tests/test_fulltextsearch.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python2.7
```

Comment:
Sorry one final request. Can you rename this file to test_full_text_search.py (insert underscores) to be consistent with full_text_search.py?